project/Dependencies.scala

```diff
@@ -122,6 +122,7 @@ object Dependencies {
     libraryDependencies ++= Seq(
       "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-spray-json" % AkkaHttpVersion,
+      "com.google.cloud" % "google-cloud-pubsub" % "0.33.0-beta",
```

Comment:
What license does this have?

Reply:
https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/LICENSE